src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -177,6 +187,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     }
     this._userInput = event.target.value;
     if (event.target.value) {
+      // tslint:disable-next-line: no-unused-expression
       this.debounceHandle;
```

Comment:
should this be `this.debounceHandle();`? Otherwise, not sure what it does

Reply:
I had to rewrite the method to actually work:
9189600